.coveragerc

```
@@ -31,4 +31,4 @@ relative_files = true
 source =
   tests
 source_pkgs =
-  awx_plugins.credentials.x.api
+  awx_plugins.credentials
```

Should we reference each module file instead? i.e. `awx_plugins.credentials.aim`

---

Maybe. I'm not sure. I haven't tested with multiple modules. But if this works, I'd leave it this way.

I'd even use
```suggestion
  awx_plugins
```
if that's enough to make it function.

Beware that Codecov's v4 action messes up some file paths, though. So don't rely on it, only on the local coverage.py which is integrated through `pytest-cov`.

And with that, I realized I forgot to integrate the HTML report generation into tox :man_facepalming:
Will do that later. But for now, running `tox -- --cov-report=html` should do the trick.

---

@chrismeyersfsu I confirmed that this works by looking into the job summary: https://github.com/ansible/awx-plugins/actions/runs/10599668926?pr=17#summary-29375340084.

A little bit of feedback, though: in the future, could you avoid hitting `Resolve` on the threads? Usually, the person starting them is most qualified to judge if they are actually solved. So I sometimes find myself trying to locate such discussions. And when they're hidden, it forces me to click on everything in the UI in hopes of finding the right one.
May I suggest just leaving a comment that it's been addressed? I've seen some folks pasting a commit link or something. If it's really straightforward, and you're sure it's okay to resolve, putting a comment is helpful. This is because clicking `Resolve` doesn't even trigger a notification. I tend to just write “I think X is solved by doing Y. Resolving this thread.” Otherwise, I'd just keep them open until the time comes to merge, at least or until the thread opener closes it.

Anyway, my point is that communication is important for good shared UX/DX.